### DISCUSSION TITLE
Init project using an already existing instance

### DISCUSSION THREAD

**Discussion by CodesInChaos** [geldata/gel#2905] [Q&A]
I have already set up a named remote instance. Now I'd like to link a project to this instance. Unfortunately `project init` seems to insist on creating a new local instance, which fails because I'm trying to run it on a windows system without docker.

How can I set up a project that uses an already existing named instance, without trying to create a new one?

**Comment by CodesInChaos** [comment]
I managed to manually create a project folder (error message when running `edgedb` without arguments outputs the expected name) containing `instance-name` and `project-path` (no symlink to the project) and it seems to work. But that's a bit inconvenient and I'm not sure if the missing link will cause problems for some features.

**Comment by fantix** [comment]
Please see https://github.com/edgedb/edgedb-cli/issues/471 and related PRs - the discussion is still ongoing, and the issue will be fixed soon.
